doc/api/next_api_changes/deprecations/24830-OG.rst

```diff
@@ -0,0 +1,5 @@
+``matplotlib.cycler``
```

Comment:
Not sure about this; seems like `matplotlib.cycler` is the more expected path.

Reply:
I will not argue against it as such, but it should be documented to be residing there in that case?

Now, searching for cycler gives a pretty clear indication.

Reply:
I suppose I get roughly the same order of hits for either one, though GitHub seems to just match the whole words and not the `.` itself, so it's hard to say if those are the exact counts.

Reply:
Sorry, incomplete message (late at night and from the phone...), I meant https://matplotlib.org/stable/search.html?q=cycler

Reply:
As far as I can tell, we only used it in a single example, so at least our documentation and code base do not seem to encourage it. (But there may historical reasons etc that I am not aware of.)

Five hits on grep.app though for matplotlib.cycler and 19 for mpl.cycler, which is quite interesting. So I may agree that this usage is maybe not that uncommon. Will revert this and add suitable comments.